pkgs/applications/science/electronics/kicad/base.nix

```
@@ -68,7 +68,8 @@ stdenv.mkDerivation rec {
 
   patches = [
     # upstream issue 12941 (attempted to upstream, but appreciably unacceptable)
-    ./writable.patch
+    ./patches/writable.patch
+    ./patches/runtime_stock_data_path.patch
```

```suggestion
    ./writable.patch
    ./runtime_stock_data_path.patch
```
we don't need to move the files into a subdirectory